docs/GettingStarted.md

```diff
@@ -73,6 +95,18 @@ Use the builder's set methods to set properties on the stream (you can read more
     builder.setFormat(oboe::AudioFormat::Float);
     builder.setChannelCount(oboe::ChannelCount::Mono);
 
+The builder's set methods can be easily chained:
```

Comment:
I don't like chaining because IMO it hinders readability and looks messy. Could just be me.

Reply:
We should at least alert developers to the option of chaining if they wish. I think it then is up to developer preference.